src/lib/middleware/assetMiddleware.ts

```diff
@@ -18,6 +18,7 @@ export function assetMiddleware() {
     return (_req, res, next) => {
       res.locals.asset = filename => {
         let manifestFile = manifest[filename] || filename
+        // TODO: Is this path still used? If no remove it.
```

Comment:
Referring to CDN_URL?

Reply:
I think this was referring to whether the Force app receives any asset requests in production at all these days. If the references all point to s3 via the CDN, it shouldn't.

Reply:
When building to prod we do looking with this CDN_URL (so this is used), but when building locally during dev force will receive requests.